src/date.js

```diff
@@ -63,3 +63,15 @@ export function withinDates (min, max, date) {
   const mDate = moment(date)
   return mDate.isBetween(min, max, 'day') || mDate.isSame(min, 'day') || mDate.isSame(max, 'day')
 }
+
+export function setDateToZeroTime(date) {
```

Comment:
@MMajko What about using [`moment().startOf('day')`](http://momentjs.com/docs/#/manipulating/start-of/) instead of this?